### ISSUE TITLE
Download links broken

### ISSUE THREAD

**Issue by saurabnigam** [rakyll/hey#222]

**Comment by abhirockzz** [comment]
@rakyll This is probably not an urgent ask by any means, but your help here would be really appreciated! Thank you!

**Comment by didier-durand** [comment]
Hi, is there another way to obtain the binaries: stored elsewhere?

**Comment by bharathnatarajan1** [comment]
@rakyll Could you please help in fixing 'Download Links', we were thinking to this great app but couldn't, due to broken links. Is there a workaround to get the binaries for Windows?

**Comment by didier-durand** [comment]
@bharathnatarajan1 : if you work on Ubuntu like me, there is a package that you can install via snapd. It works fine : look at line 225 of my https://github.com/didier-durand/knative-on-cloud-kubernetes/blob/master/.github/workflows/gcloud-gke-knative.yml to see how I did it to work on Github CI / CD

**Comment by bharathnatarajan1** [comment]
> @bharathnatarajan1 : if you work on Ubuntu like me, there is a package that you can install via snapd. It works fine : look at line 225 of my https://github.com/didier-durand/knative-on-cloud-kubernetes/blob/master/.github/workflows/gcloud-gke-knative.yml to see how I did it to work on Github CI / CD

@didier-durand Thankyou :) Is there a way to download the package in Windows machine? Need this to test FaaS app running on Windows 10 Platform.

**Comment by didier-durand** [comment]
The simplest is to install an Ubuntu instance on your Windows machine (VirtualBox, HyperV, etc.) and run hey from there.

**Comment by gnat** [comment]
Thankfully the binary is still available with a simple `apt install hey`

But still, if you no longer wish to maintain the google account, just put a /bin directory in the repository or something! @rakyll 

I feel like this broke at least a few hundred CI pipelines out in the wild, ha.

**Comment by SachinMaharana** [comment]
I couldn't install it through `sudo apt install hey`
```
sudo apt install hey                                                                            1 ↵
Alias tip: _ apt install hey
Reading package lists... Done
Building dependency tree       
Reading state information... Done

No apt package "hey"
E: Unable to locate package hey
```

If anybody still has issues with installing in ubuntu,  could use the following command:

`curl -sf https://gobinaries.com/rakyll/hey | sh`

**Comment by didier-durand** [comment]
Look at my comment above : https://github.com/rakyll/hey/issues/222#issuecomment-695233192  you have a working solution for Ubuntu and the proof that it works if you look at the execution journals.

**Comment by SachinMaharana** [comment]
@didier-durand yes sir, i did saw your solution for ubuntu, and that also works, but personally i am not a big fan of snaps , as i had noticed few weird issues in the past in my ubuntu system. using the above command works for me very well.

**Comment by didier-durand** [comment]
That's the only thing that I tried. Sorry & good luck! Didier

**Comment by kuzminets** [comment]
> If anybody still has issues with installing in ubuntu, could use the following command:
> 
> `curl -sf https://gobinaries.com/rakyll/hey | sh`

The script downloads from this link https://gobinaries.com/binary/github.com/rakyll/hey?os=linux&arch=amd64&version=v0.1.4

**Comment by raylee** [comment]
Another option is to install the [go](https://golang.org/dl/) compiler, it's quick! With that you can build the executable from source by: `go get github.com/rakyll/hey` The resulting binary will be $(go env GOPATH)/bin/hey. I.e.:

```
$ go get github.com/rakyll/hey
$ sudo cp $(go env GOPATH)/bin/hey /usr/local/bin/
$ hey
Usage: hey [options...] <url>

Options:
```
[...]

**Comment by Z01ndels** [comment]
@rakyll Have you considered moving hey releases in GitHUb ? They provide some nice tooling!
There seems to be quite complete guides like this one https://github.com/mh-cbon/go-github-release

**Comment by wahmedswl** [comment]
https://github.com/rakyll/hey/issues/222#issuecomment-704918310 is good option, downloaded build from windows via

[https://gobinaries.com/binary/github.com/rakyll/hey?os=windows&arch=amd64&version=v0.1.4](url)

**Comment by rustberry** [comment]
> Another option is to install the [go](https://golang.org/dl/) compiler, it's quick! With that you can build the executable from source by: `go get github.com/rakyll/hey` The resulting binary will be $(go env GOPATH)/bin/hey. I.e.:
> 
> ```
> $ go get github.com/rakyll/hey
> $ sudo cp $(go env GOPATH)/bin/hey /usr/local/bin/
> $ hey
> Usage: hey [options...] <url>
> 
> Options:
> [...]
> ```

For those use this method under Windows and need to use proxy for `go get`:
type below proxy setting as environment variables, then `go get` will use these settings.
```
set HTTPS_PROXY=host:port
set https_proxy=host:port
go get github.com/rakyll/hey
```

**Comment by alexellis** [comment]
I have built a builder for hey, which produces binaries to GitHub for download.

It has:
* Windows 64-bit binary
* MacOS 64-bit binary
* MacOS 64-bit binary for M1 Arm
* Linux 64-bit binary
* Linux 64-bit ARM binary
* Linux 32-bit ARM binary (Raspberry Pi)

Folks you're welcome to use that. @rakyll, if you'd like a PR, I'd be open to upstreaming this. GitHub runs the works, and there's nothing for you to do really.

https://github.com/alexellis/hey

We will probably also add it to [arkade](https://github.com/alexellis/arkade) for download via `arkade get hey`

See releases: https://github.com/alexellis/hey/releases/
